README.md

```
@@ -58,120 +72,435 @@ It powers transparent and auditable sustainability tracking within the **Stabili
+| `lib/` | Contains all external dependencies installed via `forge install` |
+| `script/` | Deployment scripts using Foundry's scripting capabilities |
+| `test/` | Unit and integration tests using Foundry's testing framework |
+
+---
+
+## Testing
+
+### Test Structure
 
-Run a local node:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)